types/jest/index.d.ts

```diff
@@ -28,7 +28,7 @@
 //                 Alexandre Germain <https://github.com/gerkindev>
 //                 Adam Jones <https://github.com/domdomegg>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.8
+// Minimum TypeScript Version: 4.5
```

Comment:
4.3, not 4.5 🙂

Reply:
```suggestion
// TypeScript Version: 4.3
```

Reply:
I simply used what `@sinclair/typebox` was referencing to 😉